openapi.yaml

```
@@ -8995,6 +8995,34 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
+  '/environment/{environmentId}/checkDockerfile':
+    post:
+      summary: Check dockerfile configuration is correct
+      operationId: checkDockerfile
+      parameters:
+        - $ref: '#/components/parameters/environmentId'
+      tags:
+        - environment
```

```suggestion
        - Environment
```
linter complaint otherwise